drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -47,6 +49,73 @@ static uint32_t get_bus_clock(uint32_t clock, uint32_t prescaler)
 	return clock / prescaler;
 }
 
+#if !defined(CONFIG_CPU_CORTEX_M4)
+
+static int32_t prepare_regulator_voltage_scale(void)
+{
+	/* Make sure to put the CPU in highest Voltage scale during clock configuration */
+	LL_PWR_ConfigSupply(LL_PWR_LDO_SUPPLY);
+	/* Highest voltage is SCALE0 */
+	LL_PWR_SetRegulVoltageScaling(LL_PWR_REGU_VOLTAGE_SCALE0);
+	return 0;
+}
+
+static int32_t optimize_regulator_voltage_scale(uint32_t sysclk_freq)
+{
+
+	/* After sysclock is configured, tweak the voltage scale down */
+	/* to reduce power consumption */
+
+	/* Needs some smart work to configure properly */
+	/* LL_PWR_REGULATOR_SCALE3 is lowest power consumption */
+	/* Must be done in accordance to the Maximum allowed frequency vs VOS*/
+	/* See RM0433 page 352 for more details */
+	LL_PWR_ConfigSupply(LL_PWR_LDO_SUPPLY);
+	LL_PWR_SetRegulVoltageScaling(LL_PWR_REGU_VOLTAGE_SCALE0);
+	return 0;
+}
+
+#if defined(CONFIG_CLOCK_STM32_PLL_SRC_HSE) || \
+	defined(CONFIG_CLOCK_STM32_PLL_SRC_HSI) || \
+	defined(CONFIG_CLOCK_STM32_PLL_SRC_CSI)
+
+static int32_t get_vco_output_range(uint32_t vco_input_range)
+{
+	if (vco_input_range == LL_RCC_PLLINPUTRANGE_1_2) {
+		return LL_RCC_PLLVCORANGE_MEDIUM;
+	}
+
+	return LL_RCC_PLLVCORANGE_WIDE;
+}
+
+static int32_t get_vco_input_range(uint32_t pllsrc_clock, uint32_t divm)
+{
+	const uint32_t input_freq = pllsrc_clock/divm;
+
+	__ASSERT(input_freq < 1000000UL || input_freq > 16000000UL,
+			"PLL1 VCO frequency input range out of range");
+
+	if (1000000UL <= input_freq && input_freq <= 2000000UL) {
+		return LL_RCC_PLLINPUTRANGE_1_2;
+
```

Comment:
remove extra blank line here and below

Reply:
Done for this line and the other ```else if``` statements.